aezeed/README.md

```diff
@@ -0,0 +1,103 @@
+# aezeed
+
+[https://github.com/lightningnetwork/lnd/pull/773](In this PR),
+we add a new package implementing the aezeed cipher
+seed scheme (based on [http://web.cs.ucdavis.edu/~rogaway/aez/](aez) ).
+
+This is a new scheme developed that aims to overcome the
+two major short comings of BIP39: a lack of a version, and a lack of a
+wallet birthday. A lack a version means that wallets may not
+necessarily know how to re-derive addresses during the recovery
+process. A lack of a birthday means that wallets don’t know how far
+back to look in the chain to ensure that they derive all the proper
+user addresses. Additionally, BIP39 use a very weak KDF. We use
+scrypt with modern parameters (n=32768, r=8, p=1). A set of benchmarks has
+been added, on my laptop I get about 100ms per attempt):
+
+```bash
+⛰ go test -run=XXX -bench=.
+
+goos: darwin
+goarch: amd64
+pkg: github.com/lightningnetwork/lnd/aezeed
+BenchmarkToMnenonic-4     	      10	 102287840 ns/op
+BenchmarkFromMnenonic-4   	      10	 105874973 ns/op
```

Comment:
While we're at it, let's update these to include the memory allocations as well. Feel free to use your own if you like, here's an output OMM:
```
goos: darwin
goarch: amd64
pkg: github.com/lightningnetwork/lnd/aezeed
BenchmarkTomnemonic-8                 20          87403684 ns/op        33559728 B/op         36 allocs/op
BenchmarkToCipherSeed-8               20          93247937 ns/op        35237697 B/op         39 allocs/op
PASS
ok      github.com/lightningnetwork/lnd/aezeed  6.388s
```